css/_components/drawer/_drawer.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="../../main.css">
+  <title>Drawer</title>
+
+  <style>
+    body {
+      display: flex;
+      flex-direction: column;
+      justify-content: space-evenly;
+      height: 400vh;
+      padding: 0 20%;
+    }
+  </style>
```

Comment:
You should avoid internal stylesheet if possible. I know this is only body and it wont matter much, but I think its practice to keep your CSS seperate

Reply:
These styles are just used as a wrapper for the components to look good for the demo part. They won't be part of the library. Hence, I just added those internally rather than adding them in the library files.